CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 - `InteractionGroups` default value for `memberships` is now `GROUP_1` (#706)
 - `ImpulseJointSet::get_mut` has a new parameter `wake_up: bool`, to wake up connected bodies.
+- Removed unmaintained `instant` in favor of `web-time`.
```

Comment:
This also needs to indicate that the wasm-bindgen dep has been removed because it's no longer needed.